pages/index.tsx

```diff
@@ -596,15 +606,15 @@ export default function Home(props: {
               Contact Us
             </a>
             <a
-              href="https://github.com/acmutd/hackportal"
+              href="https://github.com/acmutd/hackutd-ix-site"
```

Comment:
Same thing about hardcoding here. Maybe it's worth moving to a "constants" file or just making it available in the environment.